templates/artist/artist_list.html

```diff
@@ -11,86 +11,87 @@
+                                <button class="button"><i class="fa fa-map-marker"></i></button>
+                            </div>
                         </div>
-                    </div>
-                    <div class="input-group" id="my-location" {% if not lat or not lon %}style="display: none;"{% endif %}>
-                        <input class="input-group-field disabled" type="text" value="My Location" disabled />
-                        <div class="input-group-button">
-                            <button class="button">&times;</button>
+                        <div class="input-group" id="my-location" {% if not lat or not lon %}style="display: none;"{% endif %}>
+                            <input class="input-group-field disabled" type="text" value="My Location" disabled />
+                            <div class="input-group-button">
+                                <button class="button">&times;</button>
+                            </div>
                         </div>
+                        <br />
+                        <button id="location-update-button" class="button">Update</button>
+                        <button id="location-reset-button" class="button">Reset</button>
                     </div>
-                    <br />
-                    <button id="location-update-button" class="button">Update</button>
-                    <button id="location-reset-button" class="button">Reset</button>
-                </div>
-            </li>
-            <li><h6>sorted by</h6></li>
-            <li class="is-dropdown-submenu-parent opens-right">
-                <a href="#"><h6>{{ order_by.name }}</h6></a>
-                <ul class="menu is-dropdown-submenu">
-                    {% for option in sort_options %}
-                        {% if option.slug != order_by.slug %}
-                            <li><a href="?campaign-status={{ campaign_status }}&genre={{ active_genre }}&sort={{ option.slug }}"><h6>{{ option.name }}</h6></a></li>
-                        {% endif %}
-                    {% endfor %}
-                </ul>
-            </li>
-            <li><h6>.</h6></li>
-        </ul>
+                </li>
+                <li class="nav-float"><h6>sorted by</h6></li>
+                <li class="is-dropdown-submenu-parent opens-right nav-float">
+                    <a href="#"><h6>{{ order_by.name }}</h6></a>
+                    <ul class="menu is-dropdown-submenu">
+                        {% for option in sort_options %}
+                            {% if option.slug != order_by.slug %}
+                                <li><a href="?campaign-status={{ campaign_status }}&genre={{ active_genre }}&sort={{ option.slug }}"><h6>{{ option.name }}</h6></a></li>
+                            {% endif %}
+                        {% endfor %}
+                    </ul>
+                </li>
+            </ul>
+            <p class="getting-location">Getting your location...</p>
```

Comment:
This line got moved to the wrong place, it should be two lines down from here above what is now line 95.